code/game/gamemodes/cult/runes.dm

```diff
@@ -398,8 +398,11 @@ var/list/teleport_other_runes = list()
 						M << "<span class='cult'>\"I accept this meager sacrifice.\"</span>"
 		if(T.mind)
 			var/obj/item/device/soulstone/stone = new /obj/item/device/soulstone(get_turf(src))
+			stone.invisibility = INVISIBILITY_MAXIMUM //so it's not picked up during transfer_soul()

```

Comment:
incorrect you could use the verb or rightclick it
it's not LIKELY but you could